src/index.html

```diff
@@ -6,7 +6,169 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar
+    calendar--start-day
+    calendar--month-length
+    ">
+
+      <div class="
+        calendar__number
+        calendar__number--1
+        calendar__number--mon
+        "></div>
```

Comment:
calendar__day should be the classname for all these days, no need to add so many classnames to days